src/org/usfirst/frc/team2706/robot/subsystems/GearHandler.java

```diff
@@ -72,5 +79,14 @@ public boolean gearCaptured() {
         }
         return false;
     }
+        
+   // uses limit switch to help see if arm is open
+      public boolean checkArmOpen() {
```

Comment:
It looks like this was indented one too many times. (I'm viewing on mobile, so it might be wrong for me)